### PR TITLE
Adding new template of Profile Card

### PR DESCRIPTION
**Pull request by TusharKesarwani**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes:  #859

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# Checklist:

- [x] I have made this from my own
- [x] I have taken help from some online resourses 
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# ATTACH SCREEN-SHOTS / DEPLOYMENT LINK
preview

### CODE REVIEW

**Review by VidhiBhatt01 — APPROVED** [review]
https://deploy-preview-862--cool-front-end-templates.netlify.app/profile%20card/

File: Netflix Clone-2/index.html [hunk 1]
```diff
@@ -1,156 +0,0 @@
-<html>
-
-<head>
-    <meta charset="utf-8" />
```
Comment (arpit456jain) [inline Netflix Clone-2/index.html:4]: update ur branch 
u r deleting this file

**Review by arpit456jain — CHANGES_REQUESTED** [review]

**Comment by arpit456jain** [comment]
improve its UI  its very dull make it colorful

**Comment by TusharKesarwani** [comment]
Done with changes

File: Netflix Clone-2/index.html (hunk 1 above)
Comment (TusharKesarwani) [reply Netflix Clone-2/index.html]: Done with changes

**Comment by TusharKesarwani** [comment]
Preview Image
preview

**Comment by TusharKesarwani** [comment]
Thank you for merging this Pull Request.

**Pull request merged**